doc/SensorFusion/SRS/srs_eng.md

```diff
@@ -0,0 +1,385 @@
+# Software Requirements Specification (SRS)
+
+## Project Title: Sensorfusion of Localization data for Pololu Zumo32U4
+
+Author: Tobias Haeckel
+Bachelor Thesis
+
+## Table of Contents
+
+- [Software Requirements Specification (SRS)](#software-requirements-specification-srs)
+  - [Project Title: Sensorfusion of Localization data for Pololu Zumo32U4](#project-title-sensorfusion-of-localization-data-for-pololu-zumo32u4)
+  - [Version History](#version-history)
```

Comment:
Version History has been removed but the table of contents is outdated. This is a good example against the use of an extensive table of contents